content/concepts/false_positives_tuning.md

```diff
@@ -9,9 +9,25 @@ chapter: false
+CRS provides _generic_ attack detection capabilities. A fresh CRS deployment has no awareness of the web services that may be running behind it, or the quirks of how those services work. Let's describe the possible scenarios with using an image:
+
+```mermaid
+%%{init: {"themeVariables": {"quadrant1Fill": "#00ff00", "quadrant2Fill": "#ff0000", "quadrant4Fill": "#ffff00"} }}%%
+quadrantChart
+    title WAF detection quadrant
+    x-axis Allowed --> Blocked
+    y-axis Normal Traffic --> Real attack
+    quadrant-1 True Positive
+    quadrant-2 False Negative 
+    quadrant-3 Business as usual
+    quadrant-4 False Positive
+```
+
+Let's begin with describing a perfect world. In theory, CRS should _allow all valid requests_. This can be seen above as `Business as usual`. Also, CRS should only block attacks. This is a `True Positive` match, and when this happens CRS is doing its job.
+
+Two additional scenarios exist. It is possible that *genuine* transactions may cause some CRS rules to match in error, if the transactions happen to match one of the generic attack behaviors or patterns that are being detected. Such a match is referred to as a `False Positive`, or false alarm. The worse case is when there is a real attack, and CRS is not able to detect it. This is known as a `False Negative`.
 
-False positives are particularly likely to happen when operating at higher [paranoia levels]({{% ref "paranoia_levels" %}} "Page describing paranoia levels."). While paranoia level 1 is designed to cause few, ideally zero, false positives, higher paranoia levels are increasingly likely to cause false positives. Each successive paranoia level introduces additional rules, with *higher* paranoia levels adding *more aggressive* rules. As such, the higher the paranoia level is the more likely it is that false positives will occur. That is the cost of the higher security provided by higher paranoia levels: the additional time it takes to tune away the increasing number of false positives.
+False positives are particularly likely to happen when operating at higher [paranoia levels]({{% ref "paranoia_levels" %}} "Page describing paranoia levels."). While paranoia level 1 is designed to cause few, ideally zero, false positives, higher paranoia levels are increasingly likely to cause false positives. Each successive paranoia level introduces additional rules, with *higher* paranoia levels adding *more aggressive* rules. As such, the higher the paranoia level is the more likely it is that false positives will occur. There is a natural relation between trying to get zero _false negatives_ and the increase of _false positives_. That is the cost of the higher security provided by higher paranoia levels: the additional time it takes to tune away the increasing number of false positives, while trying to have less _false negatives_.
```

Comment:
```suggestion
False positives are particularly likely to happen when operating at higher [paranoia levels]({{% ref "paranoia_levels" %}} "Page describing paranoia levels."). While paranoia level 1 is designed to cause few false positives, higher paranoia levels are increasingly likely to cause false positives. Each successive paranoia level introduces additional rules, with *higher* paranoia levels adding *more aggressive* rules. As such, the higher the paranoia level is, the more likely it is that false positives will occur, and the more time must be invested to tune CRS to reduce false positives. There is no value to a system with a high paranoia level that produces many false positives, as it will likely be unusable by benign clients.
```